jinad/api/endpoints/flow.py

```diff
@@ -0,0 +1,165 @@
+import uuid
+from ruamel.yaml import YAML
+from jina.flow import Flow
+from jina.clients import py_client
+from jina.excepts import GRPCServerError
+from typing import List, Union, Optional
+from fastapi import APIRouter, Body, Response
+
+from models.pod import PodBase, PodBaseExample, PodContainer
+from excepts import FlowYamlParseException, HTTPException
+from logger import get_logger
+
+logger = get_logger(context='flow-api')
+router = APIRouter()
+flow_dict = {}
+
+
+class FlowWrapper:
+    def __init__(self, 
+                 pods: List[PodBase] = None,
+                 yaml_spec=None):
+        if pods:
+            self.f = Flow()
+            self.pods = pods
+            self._build_with_pods()
+        
+        if yaml_spec:
+            yaml = YAML()
+            yaml.register_class(Flow)
+            try:
+                self.f = yaml.load(yaml_spec)
+            except Exception as e:
+                logger.error(f'Got error while loading from yaml {e}')
+                raise FlowYamlParseException
+        
+        self.start()
+
+    def _build_with_pods(self):
+        for pod in self.pods:
```

Comment:
shouldn't we ensure that we use the `exact` same methods as the normal Flow does in a regular python client call so that we do not have discrepancies?